src/renderer/features/wallet-details/model/wallet-proxies-model.ts

```diff
@@ -19,10 +18,12 @@ export const fetchWalletProxiesFx = createEffect(
     wallet: Wallet;
     chains: Record<ChainId, Chain>;
     apis: Record<ChainId, ApiPromise>;
-  }): Promise<Record<ChainId, Omit<ProxyAccount, 'id' | 'delay'>[]>> => {
+  }): Promise<Record<ChainId, (Omit<ProxyAccount, 'id' | 'delay'> & { deposit: string })[]>> => {
     const chainIds = keys(chains);
 
-    const fetchChainProxies = async (chainId: ChainId): Promise<[ChainId, Omit<ProxyAccount, 'id' | 'delay'>[]]> => {
+    const fetchChainProxies = async (
+      chainId: ChainId,
+    ): Promise<[ChainId, (Omit<ProxyAccount, 'id' | 'delay'> & { deposit: string })[]]> => {
```

Comment:
It should rather be something like
```ts
type Proxy = Omit<ProxyAccount, 'id' | 'delay'>
type OnChainProxies = { proxies:  Proxy[], deposit: string }
```

Reply:
Fixed,
`const fetchChainProxies = async (chainId: ChainId): Promise<[ChainId, Proxy[]]>`